sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -3,6 +3,7 @@
 from itertools import count
 import warnings
 
+from math import ceil
```

Comment:
This need to be added on the top of the file (this is from the std library.)

Reply:
Done in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/5154b70788bfb1b8107303eb68e6b739e39ebfd5